migrations/versions/2018_12_05_remove_constraints.py

```diff
@@ -0,0 +1,26 @@
+"""Remove redundant constraints and rename from uix -> uq to follow naming convention
```

Comment:
This one no longer removes any redundant constraints, just rename from `uix_` to `uq_`.

Reply:
Fixed